frontend/src/pages/GamePage.tsx

```diff
@@ -1,99 +1,222 @@
-                }, Money: ${stats.money + selectedOption.impact.money}.`}
-            </TerminalOutput>,
-            <br />,
+          ...prevData,
+          "Questions are loaded. Type '/start' to begin!",
         ]);
+      } else {
+        setGameOver(true);
+        setTerminalLineData((prevData) => [...prevData, "No questions available. Game over!"]);
+      }
+    } catch (err) {
+      setError(err instanceof Error ? err.message : "An error occurred.");
+      setTerminalLineData((prevData) => [
+        ...prevData,
+        `Error: ${err instanceof Error ? err.message : "Unknown error"}`,
+      ]);
+    } finally {
+      setIsLoading(false);
+    }
+  };
+
+  const startGame = () => {
+    if (questions.length === 0) {
+      setTerminalLineData((prevData) => [
+        ...prevData,
+        "Questions are not loaded yet. Please wait...",
+      ]);
+      return;
+    }
+
+    setCurrentQuestionIndex(0); // Start with the first question
+    const firstQuestion = questions[0];
+    const questionText = `Question 1: ${firstQuestion.title}`;
+    const optionsText = firstQuestion.options
+      .map((option, index) => `${index + 1}. ${option.text}`)
+      .join("\n");
 
-        // Move to the next question
-        const nextQuestionIndex = currentQuestionIndex + 1;
-        if (nextQuestionIndex < quizQuestions.length) {
-            setCurrentQuestionIndex(nextQuestionIndex);
-            const nextQuestion = quizQuestions[nextQuestionIndex];
-            setTerminalLineData((prevData) => [
-                ...prevData,
-                <br />,
-                <TerminalOutput>{`Question ${nextQuestionIndex + 1}: ${nextQuestion.question}`}</TerminalOutput>,
-                ...nextQuestion.options.map((option, index) => (
-                    <TerminalOutput key={`t-${index}`}>{`${index + 1}. ${option.text}`}</TerminalOutput>
-                )),
-            ]);
-        } else {
-            setTerminalLineData((prevData) => [
-                ...prevData,
-                <TerminalOutput>
-                    {`Game Over! Final stats - Health: ${stats.health}, Happiness: ${stats.happiness}, Money: ${stats.money}.`}
-                </TerminalOutput>,
-            ]);
-        }
+    setTerminalLineData((prevData) => [...prevData, questionText, optionsText]);
+  };
+
+  const submitAnswer = async (
+    questionId: string,
+    selectedOption: string,
+    newStats: { health: number; wealth: number; happiness: number }
+  ): Promise<boolean> => {
+    try {
+      const response = await PanicApi.post("/submitAnswer", {
+        userId,
+        answeredQuestion: {
+          questionId,
+          selectedOption,
+        },
+        updatedStats: newStats, // Sending updated stats to backend
+      });
+
+      if (response.status === 200) {
+        return true; // Successfully submitted answer
+      } else {
+        throw new Error(response.data.message || "Failed to submit answer.");
```

Comment:
Show a toast here instead of throwing an Error, just like it is used in the `LandingPage.tsx`. We're using react-toastify for it.